README.md

```diff
@@ -1,135 +1,33 @@
-```
-$ sudo snap install rockcraft
-$ rockcraft -v
-...
-Setting the ROCK's Control Data                                                
-Control data written                                                           
-Metadata added                                                                 
-Exporting to OCI archive                                                       
-Exported to OCI archive 'ceph_0.1_amd64.rock' 
-```
-
-The freshly built container archive can be loaded into docker for subsequent operations using [skopeo](https://github.com/containers/skopeo) as:
-```
-$ sudo /snap/rockcraft/current/bin/skopeo --insecure-policy copy oci-archive:ceph_0.1_amd64.rock docker-daemon:canonical/ceph:latest
-Getting image source signatures
-Copying blob 3153aa388d02 done  
-Copying blob e3162b5ec315 done  
-Copying blob 9131ac168a8b done  
-Copying blob 0e56abe5b4e6 done  
-Copying config 4d47f598e7 done  
-Writing manifest to image destination
-Storing signatures
-```
-
-All images available locally can be checked through:
-```
-$ sudo docker images
-REPOSITORY         TAG          IMAGE ID       CREATED       SIZE
-canonical/ceph     latest       4d47f598e7ef   4 hours ago   1.51GB
-```
-
-This freshly baked Image can now be used for deploying Ceph using:
+## Using Ceph ROCKs
+The Ceph ROCK available at [GH Container Repository](https://github.com/canonical/ceph-containers/pkgs/container/ceph) can be used with popular containerised-ceph deployment tools like:
 1. [Cephadm](https://discourse.ubuntu.com/t/using-cephadm-to-deploy-custom-ubuntu-ceph-images-in-a-containerised-manner/)
 2. [Rook](https://discourse.ubuntu.com/t/deploying-ceph-with-rook/)
 
-## Automated local deployments using cephadm and lxd
-
-We also provide a python3 script which can deploy a single node ceph cluster for you to tinker with using our image, cephadm and lxd. For this to work the host should have lxd snap installed and initialised.
+## What features are supported ?
 
-```
-$ sudo snap install lxd
-$ lxd init --auto
-```
+We support 2 backends: Cephadm and Rook. The features each support differ, and this compatibility matrix aims to document what is achievable for both.
 
-### Script Usage:
-1.) Use Script to deploy a custom image:
-```
-python3 test/deploy.py image <qualified_image_name>
-```
 > **_NOTE:_**
-<qualified_image_name> can be a reference to a container image hosted on any public container registry.
-For Example:
-```
-$ python3 test/deploy.py image ghcr.io/canonical/ceph:main
-```
-2.) Use Script to clean a deployment (using script generated model file):
-```
-python3 test/deploy.py delete model-88HJ.json
-```
+A green light means that a particular feature is supported, and a red light means that it isn't. A yellow light means that a feature is _technically_ supported, but needs a considerable amount of fiddling.
```

Comment:
I realize this has just been moved but still wonder about phrasing here, maybe "needs additional manual configuration"?

Reply:
I have changed to "needs operator intervention"